app/controllers/admin/application_controller.rb

```diff
@@ -10,6 +10,17 @@ class ApplicationController < Administrate::ApplicationController
 
     before_action ->() { authorize! :manage, :all }
     before_action :complete_two_factor_setup
+    around_action :setup_logging
+
+    def setup_logging
```

Comment:
Maybe a block can't take a block as an argument? Not worth worrying about for now.